modules/lang-painless/build.gradle

```diff
@@ -96,7 +96,8 @@ tasks.named("yamlRestTestV7CompatTest").configure {
             'painless/40_fields_api/script fields api for dates',
             'painless/70_execute_painless_scripts/Execute with double field context (multi-value, fields api)',
             'painless/40_fields_api/filter script fields api',
-            'painless/40_fields_api/script score fields api'
+            'painless/40_fields_api/script score fields api',
+            'painless/70_mov_fn_agg/*' // Agg moved to a module.
```

Comment:
This is kind of a shame.

Reply:
Right, because the integration test cluster started here doesn't have the analytics module and therefor the moving average pipeline aggregation isn't available when running these bwc yaml tests. We won't have this test coverage in the aggregations module, since the tests didn't exist in this module in earlier versions (in fact the entire module doesn't exist).

We can maybe think about adding the aggregations module to the painless test cluster? Then we don't lose test coverage?